src/Sylius/Bundle/ResourceBundle/Controller/RequestConfiguration.php

```diff
@@ -268,7 +269,12 @@ public function getLimit()
      */
     public function isPaginated()
     {
-        return (bool) $this->parameters->get('paginate', true);
+        $paginate = (bool) $this->parameters->get('paginate', true);
+        $requestPaginate = $this->request->get('paginate', $paginate);
+        if(is_string($requestPaginate)){
+            $requestPaginate = filter_var($requestPaginate, FILTER_VALIDATE_BOOLEAN, FILTER_NULL_ON_FAILURE);
```

Comment:
Without the 3rd parameter, it returns `false` on failure by default (instead of `null` that is casted to `false`).